server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -539,9 +538,6 @@ protected Node(final Environment initialEnvironment, Collection<PluginInfo> clas
 
             final Settings settings = pluginsService.updatedSettings();
 
-            // Ensure to initialize Feature Flags via the settings from opensearch.yml
-            FeatureFlags.initializeFeatureFlags(settings);
```

Comment:
If plugin's introduce (can they?) a feature flag, don't we need to initialize them again?

Reply:
It doesn't seems like we allow feature flags to be directly introduced from plugin, if that's the case then we can remove it. 
Plugins ideally shouldn't be allowed to modify/override the feature flag value, so that's fine.